check-version-bump.sh

```diff
@@ -0,0 +1,25 @@
+#! /usr/bin/env bash
+
+current_version=`python setup.py --version`
+
+if git diff-index --quiet origin/master openfisca_france
```

Comment:
We should consider as “no change” changes to whitespace and `.md` files, like before. That was added for a reason.